pkg/webui/components/input/byte.js

```diff
@@ -147,10 +147,28 @@ export default class ByteInput extends React.Component {
 
   @bind
   onChange(evt) {
+    const { max, showPerChar } = this.props
+    const { data } = evt.nativeEvent
+
+    let value = clean(event.target.value)
+    const normalizedMax = showPerChar ? Math.ceil(max / 2) : max
+
+    // Check if the value already has length equal to `max`.
+    const isValueMaxLength = value.length === normalizedMax * 2
+    // Check if the cursor is placed after the last placeholder chararcter.
+    const isCursorAfterMask = evt.target.selectionStart === normalizedMax * 3 - 1
+
+    if (!isValueMaxLength && isCursorAfterMask && data !== null) {
+      const hexMatch = data.match(hex)
+      if (hexMatch !== null) {
+        value += hexMatch[0]
```

Comment:
Let's then also set the cursor position back to the end of the current incomplete value.

Reply:
Well, this is how it works now:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/16374166/91994692-8f3bb980-ed3f-11ea-9b38-f0ee81df84c8.gif)

Tested in chrome/firefox and safari.